jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/elements/common/Chip.java

```diff
@@ -15,7 +19,8 @@
  * To see an example of Chip web element please visit https://vuetifyjs.com/en/components/chips/
  */
 
-public class Chip extends UIBaseElement<ChipAssert> implements HasClick, HasIcon, HasLabel {
+public class Chip extends UIBaseElement<ChipAssert> implements HasClick, HasLabel, HasColor, HasIcon, HasImage,
+        HasTheme, IsOutlined {
 
     private static final String TEXT1 = ".v-chip__content";
```

Comment:
It's just custom implementation, please delete it from framework:
ALL_TEXTS_FROM_COMPOSITE_LABEL
BOLD_TEXT_FROM_COMPOSITE_LABEL
REGULAR_TEXT_FROM_COMPOSITE_LABEL